.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+## Contributing
+
+Hi there! We're thrilled that you'd like to contribute to this project.
+Your help is essential for keeping it great.
+
+## Before you start coding
+
+### Editing `README.md` and similar files
+
+Note that some files are managed outside this repository and are committed automatically.
+
+The `README.md` is generated automatically from our doc. If you'd like to see more details
+about something, feel free to open an issue.
+
+`.github` directory is manage in [this repository](https://github.com/algolia/algoliasearch-client-common),
+any Pull Request there is welcome.
+
+## Opening an issue
+
+Each repository provide a template for issues. Please fill the client and language version and
+provide a clear description of the problem you're facing. Steps to reproduce or example code
+(repository, jsfiddle and such) are a big help.
+
+## Submitting a pull request
+
+Keep your changes as focused as possible. If there are multiple changes you would like to make,
+please consider submitting them as separate pull requests unless they are related to each others.
+
+Here are a few tips to increase the likelihood of being merged:
+
+- [ ] Write tests.
+- [ ] Write a [good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+- [ ] Allow [edits from maintainers](https://blog.github.com/2016-09-07-improving-collaboration-with-forks/).
+
+### API Breaking changes
+
+We care deeply about backward compatibility for our API clients libraries. If it's necessary,
+we're ready to break backward compatibility but this should be pretty rare.
+
+If you want to make breaking changes, open an issue first to discuss it with the maintainers.
+
+## Label caption
+
+Labels across all repositories are normalized.
```

Comment:
Maybe `across all [Algolia] repositories`, to make it clear even to non-algolia employees?